src/components/EmailItem/index.jsx

```diff
@@ -0,0 +1,23 @@
+const EmailItem = ({ name, date, subject, body, read, id }) => {
+  let readStatus = ""
+  if (read === "0") {
+    readStatus = " border-white bg-gray-600 text-white"
+  } else {
+    readStatus = " border-gray-300 bg-gray-100 text-black"
```

Comment:
put this on line 2 in the let default value, then you can remove the else statement